hm-gtm.php

```diff
@@ -18,33 +18,59 @@
  * Output the gtm tag, place this immediately after the opening <body> tag
  *
  * @param bool $echo
+ * @param bool $network True to include network wide Google Tag Manager
+ *
  * @return string
  */
 function tag( $echo = true ) {
 
 	$id = get_option( 'hm_gtm_id', false );
 
-	if ( ! $id ) {
-		return '';
+	$tag = '';
+
+	$data_layer = Plugin::data_layer();
+
+	if ( $id ) {
+		$tag = sprintf( '
+			%2$s
+			<!-- Google Tag Manager -->
```

Comment:
There's an opportunity to avoid repeating ourselves here - the tag manager code is the same for both with the exception of what gets substituted in so best to assign the tag code to a variable and pass that to the `sprintf()` instances